src/plugin.js

```diff
@@ -1,3 +1,4 @@
+/* global DocumentFragment */
```

Comment:
Cool thanks! it just needs to be moved from a `devDep` to a `dep` like the other PR.